client/src/components/Event/EventDetail.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import { describe, it, expect, vi, afterEach } from 'vitest'
+import { render, screen, waitFor, cleanup } from '@testing-library/react'
+import '@testing-library/jest-dom/vitest'
+
+import { EventDetail } from './EventDetail'
+import { WeatherForEvent } from './EventWeather'
+import type { PollingEvent } from './availability/availability'
+
+const jsonResponse = (data: unknown, init?: ResponseInit): Response =>
+  new Response(JSON.stringify(data), {
+    status: 200,
+    headers: { 'Content-Type': 'application/json' },
+    ...init,
+  })
+
+afterEach(() => {
```

Comment:
afterEach by bylo lepsi volat uvnitr kazdeho `describe`, napr radek 22 a 87 (pred volanim it()).